Makefile

```diff
@@ -108,8 +108,10 @@ clean: ## clean up binaries
 	@echo "🐳 $@"
 	@rm -f $(BINARIES)
 
-install: $(BINARIES) ## install binaries
-	@echo "🐳 $@"
+install: ## install binaries
+	@ls $(BINARIES) > /dev/null 2>&1 || \
+		(echo "👹 Please run \`make\` before running \`make install\`." && false)
+	@echo "🐳 $@ $(BINARIES)"
 	@mkdir -p $(DESTDIR)/bin
```

Comment:
Following snippet should work without `mkdir`, but it is more complicated (and requires GNU install)
```makefile
@( for f in $(BINARIES); do install -D $f $(DESTDIR)/bin/$(basename $f); done )
```